install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -515,17 +523,13 @@ cni:
 conntrackGCInterval: ""
 
 # -- Configure container runtime specific integration.
+# -- Deprecated in favor of bpf.autoMount.enabled. To be removed in 1.15.
```

Comment:
Sorry I didn't realise we already had a comment; we don't need the double dash here, or this will be considered a different entry from the comment above when generating the docs. Apologies for the misdirection :disappointed: 

```suggestion
# Deprecated in favor of bpf.autoMount.enabled. To be removed in 1.15.
```

Reply:
no worries, fixed up.